views/error.hbs

```diff
@@ -0,0 +1,6 @@
+{{!< default }}
+<div class="sub-page-content section-wrapper">
+    <h2>Error</h2>
+    <p> {{ error }} </p>
```

Comment:
Because I was initially scared and confused, we may want to rename this to `{{ message }}` here, and above in `res.render("error", { error: err.message });`.

I originally thought it was logging an Error object/stack, which is generally a pretty :-1: user experience.